packages/amazonq/.vscode/tasks.json

```diff
@@ -44,7 +44,8 @@
             "problemMatcher": "$ts-webpack-watch",
             "options": {
                 "cwd": "${workspaceFolder}/../../packages/core"
-            }
+            },
+            "presentation": { "panel": "dedicated" }
```

Comment:
This solved the following issue:
- The webpack serve command would fail
- A subsequent build task would reuse this terminal since it failed
- The reuse caused the port in use error message to be hidden

Now the webpack serve command is in a dedicated terminal that does not disappear, so it is more obvious when it breaks.